### PR TITLE
VB comment cleanup

### PR DESCRIPTION
**Pull request by drewnoakes**
Also removes redundant numeric suffixes.

### CODE REVIEW

File: src/Microsoft.VisualStudio.AppDesigner/interop/NativeMethods.vb [hunk 1]
```diff
@@ -16,10 +16,10 @@ Namespace Microsoft.VisualStudio.Editors.AppDesInterop
...
         Public Shared ReadOnly VBCompilerGuid As Guid = New Guid(VB_COMPILER_GUID)
 
-        '/ <summary>
-        '/     Handle type for HDC's that count against the Win98 limit of five DC's.  HDC's
-        '/     which are not scarce, such as HDC's for bitmaps, are counted as GDIHANDLE's.
-        '/ </summary>
+        ' <summary>
+        '     Handle type for HDC's that count against the Win98 limit of five DC's.  HDC's
+        '     which are not scarce, such as HDC's for bitmaps, are counted as GDIHANDLE's.
+        ' </summary>
```
Comment (davkean) [inline src/Microsoft.VisualStudio.AppDesigner/interop/NativeMethods.vb]: Should be triple slash right?

Comment (drewnoakes) [reply src/Microsoft.VisualStudio.AppDesigner/interop/NativeMethods.vb]: I believe so, yes. Will update.

Comment (davkean) [reply src/Microsoft.VisualStudio.AppDesigner/interop/NativeMethods.vb]: This is spread throughout the changes not this one location.

Comment (drewnoakes) [reply src/Microsoft.VisualStudio.AppDesigner/interop/NativeMethods.vb]: The others are `<devdoc>` which are not VB standard as far as I know.

**Pull request merged**